Source/Relativity.Test.Helpers/Relativity.Test.Helpers/ArtifactHelpers/FieldHelper.cs

```diff
@@ -25,10 +25,10 @@ public class FieldHelper : IFieldsHelper
 
 		public static int GetFieldArtifactID(IServicesMgr svcMgr, string fieldName, int workspaceId)
 		{
-			if (fieldName.IsNullOrEmpty())
-			{
-				throw new ArgumentNullException(nameof(fieldName), "Field name cannot be null or empty.");
-			}
+			//if (fieldName.IsNullOrEmpty())
```

Comment:
Is there a reason for this to get commented out?

Reply:
nice catch Tom! I forgot to undo this.. I had a issue that was caused by the line but seems to have been fixed!